### DISCUSSION TITLE
How can I set the framerate of my ps3 eye to 120 fps?

### DISCUSSION THREAD

**Discussion by ticklish-bear** [opentrack/opentrack#1776] [Q&A]
In earlier releases one could manually set a framerate, however now, only 30/60 fps is available. I have the necessary drivers and everything works just as expected, just this one thing is bothering me a bit. Are there any solutions?

**Comment by GO63-samara** [comment]
Try PS3 Eye open driver: https://github.com/opentrack/opentrack/wiki/PS3-Eye-open-driver-instructions
